websocket/application.go

```diff
@@ -217,6 +217,7 @@ func PushTopic(wss *WSSession, pWatchLog *WatchLogPacket) {
 	log.Infof("stop push %s to %s", sTopic, wss.GetSessionID())
 }
 
+// isDockerImageOperationLog func that check the log whether is docker image' operating log.
```

Comment:
// isDockerImageOperationLog check the log whether is the log of pulling or pushing docker image.